.gitignore

```diff
@@ -0,0 +1,5 @@
+**/__pycache__
```

Comment:
[General] 

- Please, provide at least basic unit tests for core features of your code.
- Consider using our [template](https://github.com/network-science-lab/cookiecutter-pypackage)

Reply:
ant I've added an e2e test to double-check, that the generator works